presto-docs/src/main/sphinx/release/release-0.167.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.167
+=============
+
+General Changes
+---------------
+
+* Fix planning failure when a window function depends on the output of another window function.
+* Fix planning failure for certain aggregation with both ``DISTINCT`` and ``GROUP BY``.
+* Fix incorrect aggregation of operator summary statistics.
+* Fix a join operator issue that could cause joins that produce and filter many rows
+  to monopolize worker threads, even after the query has finished.
+* Expand plan predicate pushdown capabilities involving implicitly coerced types.
+* Short-circuit inner and right join when right side is empty.
+* Optimize constant patterns for ``LIKE`` predicates that use an escape character.
+* Validate escape sequences in ``LIKE`` predicates per the SQL standard.
+* Reduce memory usage of :func:`min_by` and :func:`max_by`.
+* Add :func:`transform_keys`, :func:`transform_values` and :func:`zip_with` lambda functions.
+* Add :func:`levenshtein_distance` function.
+* Add JMX stat for the elapsed time of the longest currently active split.
+* Add JMX stats for compiler caches.
```

Comment:
It's subjective, but these seem like something that could be useful to certain users. I don't think there is harm in putting them in the release notes.